static/app/types/workflowEngine/dataConditions.tsx

```diff
@@ -55,24 +55,32 @@ export enum DataConditionType {
   EVENT_UNIQUE_USER_FREQUENCY_WITH_CONDITIONS_PERCENT = 'event_unique_user_frequency_with_conditions_percent',
 }
 
+export enum AgeComparison {
+  OLDER = 'older',
+  NEWER = 'newer',
+}
+
 export enum DataConditionGroupLogicType {
   ANY = 'any',
   ANY_SHORT_CIRCUIT = 'any-short',
   ALL = 'all',
   NONE = 'none',
 }
 
-export interface DataCondition {
+export interface NewDataCondition {
   comparison: any;
   comparison_type: DataConditionType;
-  condition_group: DataConditionGroup;
+  condition_group?: DataConditionGroup;
+}
+
+export interface DataCondition extends Readonly<NewDataCondition> {
   condition_result: any;
   id: string;
```

Comment:
Should these also be marked `readonly`?

Reply:
fixed in https://github.com/getsentry/sentry/pull/89233/commits/c3b26dc656daeca63eb34f33b18ab0b48f40ce34 !